Gemfile

```diff
@@ -14,6 +14,6 @@ if File.exist?(local_gemfile)
 else
   gem 'activerecord', ENV.fetch('AR_VERSION', '> 5')
   gem 'mysql2', ENV.fetch('MYSQL_VERSION', '~> 0.5')
-  gem 'pg', ENV.fetch('PG_VERSION', '~> 0.2')
+  gem 'pg', ENV.fetch('PG_VERSION', '>= 0.2')
```

Comment:
This is not related to this PR, but I wasn't able to run tests locally for postgresql otherwise since ActiveRecord 7.0 requires at least `pg` version 1.1.